### PR TITLE
Fix bug in deploy.js

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

**Comment by superyyrrzz** [comment]
:+1:

File: tools/Deployment/deploy.js [hunk 1]
```diff
@@ -436,11 +436,16 @@ function updateChocoConfigPromiseFn() {
...
       if (!globalOptions.pkgName) {
         reject(new Error('package name can not be null/empty/undefined while pushing choco package'));
       }
-      return execPromiseFn('choco', ['push', globalOptions.pkgName], config.choco.homeDir)();
+      let promiseFn = execPromiseFn('choco', ['push', globalOptions.pkgName], config.choco.homeDir);
+      promiseFn().then(function () {
+        resolve();
+      }).catch(function (err) {
+        reject(err);
+      });
```
Comment (chenkennt) [inline tools/Deployment/deploy.js]: just `promiseFn().then(resolve).catch(reject)`?

**Comment by hellosnow** [comment]
:shipit:

**Comment by chenkennt** [comment]
:+1:

**Pull request merged**